Makefile

```
@@ -122,10 +121,8 @@ compile-requirements: pre-requirements $(COMMON_CONSTRAINTS_TXT) ## Re-compile *
 	@# time someone tries to use the outputs.
 	sed 's/Django<5.0//g' requirements/common_constraints.txt > requirements/common_constraints.tmp
 	mv requirements/common_constraints.tmp requirements/common_constraints.txt
-	sed 's/pip<24.3//g' requirements/common_constraints.txt > requirements/common_constraints.tmp
+	sed 's/pip<25.3//g' requirements/common_constraints.txt > requirements/common_constraints.tmp
 	mv requirements/common_constraints.tmp requirements/common_constraints.txt
-	pip-compile -v --allow-unsafe ${COMPILE_OPTS} -o requirements/pip.txt requirements/pip.in
-	pip install -r requirements/pip.txt
 
 	pip-compile -v ${COMPILE_OPTS} -o requirements/pip-tools.txt requirements/pip-tools.in
```

I believe we'll need `--allow-unsafe` in order to get a `pip` dep in the `pip-tools.txt` lockfile (and then run `make compile-requirements` to rebuild). Currently it's excluded.

```suggestion
	pip-compile -v --allow-unsafe ${COMPILE_OPTS} -o requirements/pip-tools.txt requirements/pip-tools.in
```

(Can't test locally, having some issues with GH at the moment.)

---

Yea, you're right, good catch.